src/applications/gi/components/profile/EstimateYourBenefitsForm.jsx

```diff
@@ -117,8 +117,8 @@ class EstimateYourBenefitsForm extends React.Component {
       this.setState({ inputUpdated: false });
       this.props.updateEstimatedBenefits();
       setTimeout(() => {
-        scroller.scrollTo(accordionId, getScrollOptions());
-        focusElement(`#${accordionId}`);
+        scroller.scrollTo(accordionButtonId, getScrollOptions());
```

Comment:
I'm not certain we need the scroller behavior here. In testing, it pushes the Estimated Benefits panel off the screen a bit. Can we remove/comment out this line and retest locally before merging?